aws-android-sdk-sagemaker-runtime/src/main/java/com/amazonaws/services/sagemakerrts/AmazonSageMakerRuntime.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.services.sagemakerrts;
```

Comment:
suggestion to rename package as sagemakerruntime

Reply:
Changed